### DISCUSSION TITLE
Insert new Record from map

### DISCUSSION THREAD

**Discussion by DblK** [pocketbase/pocketbase#3229] [Q&A] [answered]
Hi,

I read the [doc here](https://pocketbase.io/docs/go-records/#create-new-record-without-data-validations) but I do not like to be forced to use `record.Set` for all fields I have.

Is it possible to give a `map` with some field/value and to save them.
I tried `SetExpand` but does not seems to work.

Thx for the help!

**Comment by ganigeorgiev** [comment, marked as answer]
`SetExpand` is for the record's "expand" data.

For the schema fields you can try with [`record.Load(data)`](https://pkg.go.dev/github.com/pocketbase/pocketbase/models#Record.Load).

**Reply by DblK** [reply]
Thank for the reply, I will make a PR on doc to illustrate this case also if you don't mind.

**Reply by ganigeorgiev** [reply]
_I'll update it sometime later today._
